builtin/mainmenu/tab_online.lua

```diff
@@ -65,13 +65,15 @@ local function get_formspec(tabview, name, tabdata)
 	end
 
 	local retval =
-		-- Search
+	-- Search
```

Comment:
Ideally *try* to keep such unrelated changes ("noise") out of PRs. I don't mind a low level of noise / cleaning up adjacent code but it's better to do cleanups separately.